firepit/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "firepit",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "fmt": "prettier --write *.js",
+    "pkg": "pkg -c package.json firepit.js --out-path dist/ && chmod +x dist/firepit-*",
+    "ship": "gsutil -m cp dist/* gs://fir-tools-builds/firepit/ && gsutil iam ch allUsers:objectViewer gs://fir-tools-builds"
+  },
+  "author": "",
+  "license": "MIT",
```

Comment:
I see other licenses (Apache 2.0) in the headers of files. This should probably match that.